ncm-authconfig/src/main/pan/components/authconfig/sssd/ldap.pan

```diff
@@ -8,14 +8,17 @@
 
 declaration template components/authconfig/sssd/ldap;
 
-type ldap_schema = string with match(SELF, "^(IPA|AD|rfc2307|rfc2307bis)") ||
-    error ("LDAP schema must be valid according to sssd-ldap: " + SELF);
+type ldap_schema = string with match(SELF, "^(IPA|AD|rfc2307|rfc2307bis)") || error(
```

Comment:
I know this is supposed to be just clean up but there's a bug here, missing $ at end.

Reply:
It could also be migrated to `choice`, maybe I'll do another set of PRs for that sort of thing.

Reply:
Looking at the this there's deeper issues, do I'll open a bugfix PR.